merlin/systems/dag/ops/tensorflow.py

```diff
@@ -143,38 +116,69 @@ def _export_model(self, model, name, output_path, version=1):
         output_path:
             The path to write the exported model to
         """
-        tf_model_path = os.path.join(output_path, str(version), "model.savedmodel")
         config = model_config.ModelConfig(
             name=name, backend="tensorflow", platform="tensorflow_savedmodel"
         )
 
+        config.parameters["TF_GRAPH_TAG"].string_value = "serve"
+        config.parameters["TF_SIGNATURE_DEF"].string_value = "serving_default"
+
+        for _, col_schema in self.input_schema.column_schemas.items():
+            _add_model_param(config.input, model_config.ModelInput, col_schema)
+
+        for _, col_schema in self.output_schema.column_schemas.items():
+            _add_model_param(config.output, model_config.ModelOutput, col_schema)
```

Comment:
This logic would likely also need to be used in `PipelineableInferenceOperator.export` https://github.com/NVIDIA-Merlin/systems/blob/572dbf154015924106718d21f1896037d8a3bab6/merlin/systems/dag/ops/operator.py#L259-L274